test_data_studies/commands.sh

```diff
@@ -0,0 +1,75 @@
+#!/bin/bash -xe
+
+export PGDATESTYLE="ISO, MDY"
+
+studies=`ls test_data_studies/studies`
+
+echo "DROPPING ENV... "
+qiita_env drop --no-ask-for-confirmation
+echo "Ok"
+
+echo "MAKING ENV... "
+qiita_env make --add-demo-user --no-load-ontologies
+echo "Ok"
+
+# Inserting all the information for each study
+echo "INSERTING STUDIES"
+mkdir -p temp
+base_id=0
+for i in ${studies[@]}; do
```

Comment:
This will not be possible to run more than once on the same system without re-downloading some files, right? Will it be possible to fix that?